docs/source/pages/widgets/widgets.rst

```diff
@@ -0,0 +1,86 @@
+=======
+Widgets
+=======
+
+On this page you can find information about our available widgets.
+
+Simple file send widget
+=======================
+
+.. image:: pic_widgets/simpleSend1.png
+   :width: 600
+   :align: center
+
+This widget allows anyone to send a file to be signed on our platform without needing to authorize on it. To send a file just drag & drop it onto the upload zone, or click it to select a file manually.
+
+.. image:: pic_widgets/simpleSend2.png
+   :width: 600
+   :align: center
+
+In the following step signers email address should be entered.
+
+.. image:: pic_widgets/simpleSend3.png
+   :width: 600
+   :align: center
+
+After it, senders email should be entered. Please note that this email address should not be registered on the platform.
+
+.. image:: pic_widgets/simpleSend4.png
+   :width: 600
+   :align: center
+
+After entering senders address, please confirm the action by entering a code that will be sent to senders email.
+
+.. image:: pic_widgets/simpleSend5.png
+   :width: 600
+   :align: center
+
+After a successful confirmation, envelope will be created and sent to the signer. Upon the envelope completion an email notification will be sent to the sender.
+
+.. Tip:: Please note, that widget has a limit of 3 files per 5 minutes from one IP address to avoid spam.
+
+To embed this widget in any web page, please do as described below:
+
+1. Insert javascript into your webpage just after a <body> tag:
```

Comment:
Add information that user must replace platform_url_here with FE url. And add example with docstudio host